src/test/java/core/basesyntax/StorageServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import model.Position;
+import model.Storage;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import services.ReportService;
+import services.StorageService;
+import java.io.File;
+import java.time.LocalDate;
+
+public class StorageServiceTest {
```

Comment:
why are you testing both StorageService and ReportService in the same class?

Reply:
done